app/email.py

```diff
@@ -1,21 +1,16 @@
-from threading import Thread
-
-from flask import current_app, render_template
+import os
+from flask import render_template
 from flask.ext.mail import Message
 from . import mail
+from app import create_app
 
 
-def send_async_email(app, msg):
+def send_email(recipient, subject, template, **kwargs):
+    app = create_app(os.getenv('FLASK_CONFIG') or 'default')
```

Comment:
Ah somewhat makes sense. I'll look more into the semantics of this when I read up on task queues. And I don't really want to look into another emailing library. Flask-mail is pretty solid...unless of course we want to switch to something from SendGrid?